wfe/web-front-end_test.go

```diff
@@ -439,7 +439,7 @@ func TestIssueCertificate(t *testing.T) {
 	})
 	test.AssertEquals(t,
 		responseWriter.Body.String(),
-		"{\"type\":\"urn:acme:error:malformed\",\"detail\":\"Error unmarshaling certificate request\"}")
+		`{"type":"urn:acme:error:malformed","detail":"Unable to read/verify body"}`)
```

Comment:
This is no longer testing the same thing; this is supposed to be a cert request with an invalid body, but now it's effectively just testing that the resource isn't set.